plugins/obs-filters/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-project(obs-filters)
+﻿project(obs-filters)
```

Comment:
You seem to have introduced a byte order mark here.  The other files also seem to have them.  You should remove them by encoding the files as UTF-8 without BOM.